### ISSUE TITLE
[Bug] admin tools image: Missing server-versioned docker tag for 1.24.x

### ISSUE THREAD

**Issue by chlunde** [temporalio/docker-builds#222]
### What are you really trying to do?

We're trying to easily use a compatible version of the temporal CLI for a given version of the temporal cluster (via temporal-operator).

### Describe the bug

| repo | status |
|---|---|
| temporalio/server:1.23.0 | :1st_place_medal: |
| temporalio/server:1.23.1 | :1st_place_medal: |
| temporalio/server:1.24.2 | :1st_place_medal: |
| temporalio/admin-tools:1.23.0 | :1st_place_medal: |
| temporalio/admin-tools:1.23.1 | :1st_place_medal: |
| temporalio/admin-tools:1.24.0 | :x: |
| temporalio/admin-tools:1.24.2 | :x: |

It seems 1.24.x uses a new tagging strategy (`temporalio/admin-tools:1.24.2-tctl-1.18.1-cli-0.13.0`), but it would be very helpful if there also existed a tag for a "compatible CLI tool for that version of temporal" using the server tag only.

### Minimal Reproduction

```bash
docker pull temporalio/admin-tools:1.24.2
```

### Environment/Versions

- Temporal Version: 1.24.2

**Comment by jmcnevin** [comment]
Agreed.  The current tagging strategy for the admin-tools container is... a lot.  It would be nice to have at least one of these images tagged as a default for a particular Temporal version.  I'm just trying to run some database migrations with it, so I don't really care about the CLI version.

**Comment by robholland** [comment]
We now tag an admintools image per release again.

**Comment by Tadimsky** [comment]
@robholland it looks like there is a `1.27` tag but not a `1.27.2` tag.
The least specific tag that pins the Temporal release version is `1.27.2-tctl-1`

Is there a chance we can get a clean `1.27.2` tag?

<img width="950" alt="Image" src="https://github.com/user-attachments/assets/143a39ad-0cd2-465b-91ac-d51c7e69233b" />
